numpy/core/tests/test_records.py

```diff
@@ -416,6 +416,21 @@ def test_pickle_3(self):
             assert_(pa.flags.writeable)
             assert_(pa.flags.aligned)
 
+    def test_pickle_void(self):
+        # issue gh-13593
+        dt = np.dtype([('obj', 'O')])
```

Comment:
Can you give this a second field too, just to be sure?

Reply:
ok